src/etl/ETLSource.cpp

```diff
@@ -673,6 +674,8 @@ class AsyncCallData
                         request_.ledger().sequence(),
                         std::string{obj.key()});
                 lastKey_ = obj.key();
+                backend.writeNFTs(std::move(getNFTDataFromObj(
+                    request_.ledger().sequence(), obj.key(), obj.data())));
```

Comment:
what about pg?

Reply:
Postgres (if that's what you mean) support is no longer a thing (see develop branch) so you don't need to think about it no more :)